test/test_exporter.py

```diff
@@ -521,6 +523,63 @@ def test_view(self, mock_post):
         # shut down cleanly to avoid failed exports later.
         meter_provider.shutdown()
 
+    def test_configuration_default(self):
+        with patch.object(PeriodicExportingMetricReader,
+                          "__init__") as mock_reader:
+            with patch.object(_DynatraceMetricsExporter,
+                              "__init__") as mock_exporter:
+                mock_reader.return_value = None
+                mock_exporter.return_value = None
+                self.assertIsInstance(configure_dynatrace_exporter(),
+                                      PeriodicExportingMetricReader)
+                mock_exporter.assert_called_once_with(
+                    endpoint_url=None,
+                    api_token=None,
+                    prefix=None,
+                    default_dimensions=None,
+                    export_dynatrace_metadata=False
+                )
+                mock_reader.assert_called_once_with(
+                    export_interval_millis=None,
+                    preferred_temporality=_DYNATRACE_TEMPORALITY_PREFERENCE,
+                    exporter=mock.ANY,
+                )
+                _, kwargs = mock_reader.call_args
+                self.assertIsInstance(kwargs.get("exporter"),
+                                      _DynatraceMetricsExporter)
+
+    def test_configuration_custom(self):
+        with patch.object(PeriodicExportingMetricReader,
+                          "__init__") as mock_reader:
+            with patch.object(_DynatraceMetricsExporter,
+                              "__init__") as mock_exporter:
+                mock_reader.return_value = None
+                mock_exporter.return_value = None
+                self.assertIsInstance(configure_dynatrace_exporter(
+                    endpoint_url="endpoint.url",
+                    export_dynatrace_metadata=True,
+                    export_interval_millis=100,
+                    api_token="dt.APItoken",
+                    prefix="otel.python.test",
+                    default_dimensions={"defaultKey": "defaultValue"}
+                ),
+                                      PeriodicExportingMetricReader)
```

Comment:
```suggestion
                    PeriodicExportingMetricReader)
```
I think?

Reply:
fixed in [36cb2d6](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/27/commits/36cb2d69d842f799aec8d23cf826184c545a7557).